specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -149,6 +149,9 @@
             "$ref": ".//examples//SuccessfulAnalyzeStatusRequest.json"
           }
         },
+        "x-ms-pageable": {
+          "operationName": "AnalyzeStatusNextPage"
```

Comment:
missing nextLinkName and itemName

Reply:
thanks Shawn. Interesting... .NET doesn't need it. At the same time, adding it didn't affect .NET so I will update the PR

Reply:
Added. Could you try again ?